deploifai/dataset/create.py

```diff
@@ -134,3 +136,20 @@ def create(context: DeploifaiContextObj):
         azure_account = cloud_data["cloudProviderYodaConfig"]["azureConfig"]["storageAccount"]
         click.secho("account-name: {}".format(azure_account))
         click.secho("container-name: {}".format(cloud_name))
+
+    # create a dataset directory locally, along with dataset.cfg file within
+    try:
+        os.mkdir(storage_name)
+    except OSError:
+        click.secho("An error occured when creating the dataset locally", fg="red")
```

Comment:
should probably add another echo after this error message saying `But the dataset has been created on Deploifai`